blueocean-dashboard/src/main/js/credentials/git/GitCredentialsPickerPassword.tsx

```diff
@@ -0,0 +1,278 @@
+import * as React from 'react';
+import {Component} from 'react';
+import {observer} from 'mobx-react';
+import {i18nTranslator} from '@jenkins-cd/blueocean-core-js';
+import {GitPWCredentialsManager, ManagerState} from './GitPWCredentialsManager';
+import * as debounce from 'lodash.debounce';
+
+import {Button} from '../../creation/github/Button';
+
+import {
+    FormElement,
+    PasswordInput,
+    TextInput,
+    RadioButtonGroup,
+} from '@jenkins-cd/design-language';
+
+const t = i18nTranslator('blueocean-dashboard');
+
+interface Props {
```

Comment:
This interface needs to be exported. and I think it should be namespaced, so that people do `GitCredentialsPickerPassword.Props`. Not totally sure of conventions here, but it seems like a good idea to me

Reply:
Just looked at some code @kzantow has done and he is not exporting props, but i think its probably not a bad standard practise. What do you guys think?